### ISSUE TITLE
Add Chat

### ISSUE THREAD

**Issue by NTaylorMullen** [NTaylorMullen/ShootR#22]
Integrate with Jabbr's server potentially.

Maybe have "chat" bubbles above ships?

It'd also be nice to have a "Guest" mode so we don't require users to login.

**Comment by AdrianTP** [comment]
I agree with this idea. I played this game for several hours straight last night because I was having fun, and I could see new players getting frustrated and leaving. I'd like to be able to say "hello" and offer to teach them how to survive. Though perhaps I'm just unique in my interest in this game, and most people aren't as attracted to it as I...

**Comment by NTaylorMullen** [comment]
Right now in the dev branch there's a "temporary" chat system.  I've been re-writing a lot of the underlying engine to ShootR to support WebGL so I can add a lot more features without having to worry about performance (perf client side that is, server side is where I want it).
